medusa/tv.py

```diff
@@ -2411,13 +2411,14 @@ def load_from_indexer(self, season=None, episode=None, tvapi=None, cached_season
                            (id=self.show.indexerid, show=self.show.name,
                             ep=episode_num(season, episode), status=statusStrings[self.status].upper()), logger.DEBUG)
 
-        # if we have a media file then it's downloaded
+        #  We only change status of the episode with media is present when status is not SNATCHED|DOWNLOADED|ARCHVED
```

Comment:
This should be "We only change the episode's status if a file exists and the status is not SNATCHED|DOWNLOADED|ARCHVED"